src/main/java/seedu/address/model/EduMate.java

```diff
@@ -52,12 +53,18 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the person list with an empty list of persons.
+     */
+    public void resetPersons() {
+        this.persons.setPersons(new ArrayList<>());
+    }
```

Comment:
Added because Sort Command requires a full flush of the Persons List.

Reply:
LGTM